dev/com.ibm.ws.beanvalidation.v20.cdi/resources/com/ibm/ws/beanvalidation/v20/cdi/internal/BVNLSMessages.nlsprops

```diff
@@ -0,0 +1,46 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# DO NOT TRANSLATE "\n" and "\t" characters.  these are for textual formatting.
+# REMINDER: if message has substitution parameters, then any apostrophe in the message text
+#           must be followed by a 2nd apostrophe to work correctly.
+# -------------------------------------------------------------------------------------------------
+#
+# Briefly:
+#     MESSAGE_ID=<ComponentPrefix><MessageNumber><TypeCode>: <MessageText>
+#
+# Message numbers are four digit numbers, from '0000' to '9999', inclusive.
+# Message numbers are filled to the left with '0', to a width of 4.
+#
+# Valid message type codes are:
+#     I (Informational)
+#     W (Warning)
+#     E (Error)
+#
+# Note the ": " that appears between the type code and the message text.
+#
+# -----------------------------------------------------------------------
+# cmvc_en_version_level=
+#CMVCPATHNAME ws/code/beanvalidation/src/com/ibm/ws/beanvalidation/resources/nls/BVNLSMessages.nlsprops
+#COMPONENTPREFIX CWNBV
+#COMPONENTNAMEFOR CWNBV Bean Validation
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_VAR
+
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} the application name
+#    {1} the exception message from trying to register the CDI extension
+# -----------------------------------------------------------------------
+UNABLE_TO_REGISTER_WITH_CDI=CWNBV0200W: Unable to register Hibernate Validator CDI extension for application {0}. As a result, CDI integration for Bean Validation will not be available for this application. The issue registering the CDI extension was: {1}
```

Comment:
The {1} issue cannot register the Hibernate Validator CDI extension for the {0} application, so CDI integration for Bean Validation is not available for this application.

Reply:
We need to keep {1} at the end of the message, since it is going to be a very long error message that comes from code we do not control and contains a full stack trace.  An example of what {1} might be is something like:

```
javax.validation.ValidationException: HV000106: org.hibernate.beanvalidation.tck.tests.xmlconfiguration.constraintdeclaration.propertylevel.User does not contain the property foobar.
	at org.hibernate.validator.internal.xml.ConstrainedGetterBuilder.findGetter(ConstrainedGetterBuilder.java:129)
	at org.hibernate.validator.internal.xml.ConstrainedGetterBuilder.buildConstrainedGetters(ConstrainedGetterBuilder.java:64)
	at org.hibernate.validator.internal.xml.MappingXmlParser.processBeanType(MappingXmlParser.java:260)
	at org.hibernate.validator.internal.xml.MappingXmlParser.parse(MappingXmlParser.java:158)
	at org.hibernate.validator.internal.metadata.provider.XmlMetaDataProvider.<init>(XmlMetaDataProvider.java:48)
	at org.hibernate.validator.internal.engine.ValidatorFactoryImpl.<init>(ValidatorFactoryImpl.java:204)
	at org.hibernate.validator.HibernateValidator.buildValidatorFactory(HibernateValidator.java:38)
	at org.hibernate.validator.internal.engine.ConfigurationImpl.buildValidatorFactory(ConfigurationImpl.java:344)
	at org.hibernate.validator.cdi.ValidationExtension.<init>(ValidationExtension.java:119)
	at com.ibm.ws.beanvalidation.v20.cdi.internal.LibertyHibernateValidatorExtension.delegate(LibertyHibernateValidatorExtension.java:102)
	at com.ibm.ws.beanvalidation.v20.cdi.internal.LibertyHibernateValidatorExtension.beforeBeanDiscovery(LibertyHibernateValidatorExtension.java:122)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jboss.weld.injection.StaticMethodInjectionPoint.invoke(StaticMethodInjectionPoint.java:95)
	at org.jboss.weld.injection.MethodInvocationStrategy$SpecialParamPlusBeanManagerStrategy.invoke(MethodInvocationStrategy.java:144)
	at org.jboss.weld.event.ObserverMethodImpl.sendEvent(ObserverMethodImpl.java:330)
	at org.jboss.weld.event.ExtensionObserverMethodImpl.sendEvent(ExtensionObserverMethodImpl.java:123)
	at org.jboss.weld.event.ObserverMethodImpl.sendEvent(ObserverMethodImpl.java:308)
	at org.jboss.weld.event.ObserverMethodImpl.notify(ObserverMethodImpl.java:286)
```

Reply:
Oh, I see. How about this?
The issue cannot register the Hibernate Validator CDI extension for the {0} application, so CDI integration for Bean Validation is not available for this application. The following issue registered the CDI extension: {1}

Reply:
That doesn't really make sense because "the issue" is not the thing trying to register the CDI extension, the server is trying to register the CDI extension and it ran into an issue when trying to do so.  How about this?
> Liberty cannot register the Hibernate Validator CDI extension for the {0} application, so CDI integration for Bean Validation will not be available for this application. The following issue occurred while registering the CDI extension: {1}

Reply:
Since we try to avoid using product names in messages, how about this...

The Hibernate Validator CDI extension for the {0} application cannot be registered, so CDI integration for Bean Validation is not available for this application. The following issue occurred while the CDI extension was registered: {1}

Reply:
Sounds good to me.  Just one minor tweak that saying "was registered" makes it sound like it was successfully registered when it wasn't.  So instead I'm going to say "was being registered"